src/recensio/plone/behaviors/pages_of_presented_text.py

```diff
@@ -31,10 +31,27 @@ class IPagesOfPresentedText(model.Schema):
     # but I have to think more about it
     directives.mode(heading__page_number_of_presented_text_in_print="display")
 
+    fieldset(
+        "reviewed_text",
+        label=_("label_schema_reviewed_text", default="Reviewed Text"),
+        fields=[
+            "heading__page_number_of_presented_text_in_print",
```

Comment:
This heading field belongs to the page start/end fields, so it should be in the same fieldset.

Reply:
the original code places it in the `reviewed_text`. is that an error and I should put it in the same fieldset, right? https://github.com/syslabcom/recensio.contenttypes/blob/master/recensio/contenttypes/content/schemata.py#L597